DOCS.md

```diff
@@ -11,440 +11,397 @@ Version 1.0.0
-“channel code” while TrueX implemented code will be referred to as
-“renderer code”.
+For simplicity, publisher implemented code will be referred to as "channel code" while true\[X\] implemented code will be referred to as "renderer code".
 
-TrueX will provide a Roku `TruexLibrary` component library that can be
-loaded into the app. This library will offer a component,
-`TruexAdRenderer`, that will need to be instantiated, initialized and
-given certain commands (described below in [TruexAdRenderer Input Events](#truexadrenderer-input-events)) by the channel
-code.
+true\[X\] will provide a Roku `TruexLibrary` component library that can be loaded into the channel. This library will offer a component, `TruexAdRenderer`, that will need to be instantiated, initialized and given certain commands (described below in [TruexAdRenderer Input Events](#truexadrenderer-input-events)) by the channel code.
 
-At this point, the renderer code will take on the responsibility of
-requesting ads from TrueX server, creating the native UI for the TrueX
-choice card and interactive ad unit, as well as communicating events to
-the channel code when action is required.
+At this point, the renderer code will take on the responsibility of requesting ads from true\[X\] server, creating the native UI for the true\[X\] choice card and interactive ad unit, as well as communicating events to the channel code when action is required.
 
-The channel code will still need to parse out the SSAI ad response,
-detect when a TrueX ad is supposed to display, pause the stream,
-instantiate `TruexAdRenderer` and handle any events emitted by the
-renderer code.
+The channel code will still need to parse out the SSAI ad response, detect when a true\[X\] ad is supposed to display, pause the stream, instantiate `TruexAdRenderer` and handle any events emitted by the renderer code.
+
+It will also need to handle skipping ads in the current ad pod, if it is notified to do so.
 
-It will also need to handle skipping ads in the current ad pod, if it is
-notified to do so.
 
 ## Product Flows
 
-There are two distinct product flows supported by `TruexAdRenderer`:
-Sponsored Stream (full-stream ad-replacement) and Sponsored Ad Break
-(mid-roll ad-replacement).
+There are two distinct product flows supported by `TruexAdRenderer`: Sponsored Stream (full-stream ad-replacement) and Sponsored Ad Break (mid-roll ad-replacement).
 
-In a Sponsored Ad Break flow, once the user hits a mid-roll break with a
-TrueX tag flighted, they will be shown a "choice-card" offering them the
-choice between watching a normal set of video ads or a fully interactive
-TrueX ad:
+In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true\[X\] tag flighted, they will be shown a "choice-card" offering them the choice between watching a normal set of video ads or a fully interactive true\[X\] ad:
 
 ![choice card](media/choice_card.png)
 
 ***Fig. A** example true\[X\] mid-roll choice card*
```

Comment:
Super nit, some markdown highlighters seem to handle nested asterisks incorrectly. Could convert the italics to use the `_` markup like so
Source: ```_**Fig. A** example true\[X\] mid-roll choice card_```
Renderer: _**Fig. A** example true\[X\] mid-roll choice card_

Reply:
good idea, thanks